src/renderer/src/css/global.css

```diff
@@ -0,0 +1,54 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@font-face {
+  font-family: "Nunito";
+  src: url("@renderer/assets/fonts/Nunito-Variable.ttf") format("truetype");
+  font-weight: 100 900;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Nunito";
+  src: url("@renderer/assets/fonts/Nunito-Variable-Italic.ttf") format("truetype");
```

Comment:
The name of the file is "Nunito-Italic-Variable.ttf" not "Nunito-Variable-Italic".